CHANGELOG.md

```diff
@@ -12,6 +12,15 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 ### Removed Features
 
 ### Deprecated Features
+- ROX-12620: We continue to simplify access control management by grouping some permissions in permission sets. As a result:
+  - The permission `WorkflowAdministration` will deprecate the permissions `Policy, VulnerabilityReports`.
+
+### Required Actions
```

Comment:
I think we are making the changelog complex again. This is related to the deprecation above so we should move it up and keep it together.

Reply:
@md2119 the reason why we've explicitly called this out in a separate section was feedback we received (i.e. this [thread in GChat](https://chat.google.com/room/AAAA9gJI6o4/M6wkSZdgp58) and from other places in solution-eng)
Essentially, we got the feedback that while the deprecation notice is nice, it does lack concrete steps of _what_ customers have to do to remediate certain things (i.e. review their permission sets). We've decided to create this section to put more highlight on these kind of things and call them out explicitly in both the changelog and the release notes afterwards.

Reply:
Fair enough.

Will close my thoughts here.

In the release notes, the required actions associated with deprecations go in the "Deprecated features". The immediate next question that customers/users ask when they see a deprecation is "what do I need to do for this deprecation?" and having that piece of information right there makes it more accessible. So basically closly tieing it up within changelog. 

Refer to example: https://docs.openshift.com/acs/3.72/release_notes/372-release-notes.html#deprecated-features-372.